FBSDKCoreKit/FBSDKCoreKit/FBSDKApplicationDelegate.m

```diff
@@ -420,6 +420,11 @@ - (void)_logIfAppLinkEvent:(NSURL *)url
     return;
   }
 
+  NSString * targetURLString = applinkData[@"target_url"];
+  if (targetURLString == nil || ![targetURLString isKindOfClass: [NSString class]]) {
+    return;
+  }
+
   NSURL *targetURL = [NSURL URLWithString:applinkData[@"target_url"]];
```

Comment:
Hi, @NicholasTD07 
I'm experiencing this crash, currently.

In my opinion, it would be better if code is like below:

NSString * targetURLString = applinkData[@"target_url"];
if (![targetURLString isKindOfClass: [NSString class]]) {
     return;
}

NSURL *targetURL = [NSURL URLWithString:targetURLString];